src/Core/Cook/Cook.cpp

```diff
@@ -10,45 +10,37 @@
 #include "Cook.hpp"
 #include <thread>
 
+extern "C" {
```

Comment:
useless

Reply:
unistd.h is in C, so it need to be encalpsulate